### ISSUE TITLE
Web Server: duplicate flow parameters after closing the popup.

### ISSUE THREAD

**Issue by armisael** [azkaban/azkaban#1481]
This is a low priority bug, but most likely an easy one to fix, and quite disturbing sometimes: basically, when you `Prepare execution`, close the popup, and open it again, overridden properties appear duplicated in the `Flow parameters` tab.

### How to reproduce it
1) Open any flow, and click `Schedule / Execute Flow`;
1) Select the `Flow parameters` tab;
1) Override a property of your choice (say `foo = bar`);
1) Run the flow
1) Once execution is completed (either with a success or with a failure), open it and click `Prepare Execution`;
1) Close the popup by either hitting `Esc` or clicking the `Cancel` button;
1) Click on `Prepare Execution` again;
1) Select the `Flow parameters` tab.

Expected behaviour: the `foo = bar` overridden parameter appears only once.
Actual behaviour: it appears twice.

**Comment by HappyRay** [comment]
Thanks for reporting this issue and the detailed repro steps!

Are you interested in fixing this issue? Please see the updated dev guide for instructions.
https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md

**Comment by armisael** [comment]
Hi,

I worked on this today, and managed to write a fix. I'll push it later tomorrow, after a few more checks.

I also took the chance to fix another small bug on the flow parameters input, I hope it won't be a problem if I push them together.

**Comment by HappyRay** [comment]
Cool. Thanks.

I would suggest a separate PR for unrelated changes. It will increase your chance of getting a quicker review.

**Comment by armisael** [comment]
Sure, I'll open another bug then, and submit two PRs. Thanks

**Comment by HappyRay** [comment]
Thanks!
